taxonomies/labels.txt

```diff
@@ -3115,19 +3115,22 @@ it:AT-BIO-301
 nl:AT-BIO-301
 pl:AT-BIO-301
 pt:AT-BIO-301
+xx:AT-BIO-301
 country:en:Austria
 
 <en:EU Organic
 en:AT-BIO-302
 de:AT-BIO-302
 fi:AT-BIO-302
+xx:AT-BIO-302
 country:en:Austria
 
 <en:EU Organic
 en:AT-BIO-401
 ca:AT-BIO-401
 fi:AT-BIO-401
 it:AT-BIO-401
+xx:AT-BIO-401
```

Comment:
Looks good.

If you add a xx: you can remove the other entries except the canonical en: entry.

<en:EU Organic
en:AT-BIO-401
xx:AT-BIO-401

Reply:
Okay I thought there is sth about the Robotoff (and others) why it should be kept.

Reply:
@stephanegigandet Just give me a short reply so I know how to proceed.